src/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -734,8 +734,16 @@ internal ProtocolToken NextMessage(byte[] incoming, int offset, int count)
             }
 
             ProtocolToken token = new ProtocolToken(nextmsg, status);
+
             if (NetEventSource.IsEnabled)
+            {
+                if (token.Failed)
+                {
+                    NetEventSource.Info(this, $"Authentication failed. Status: {status.ToString()}, Exception message: {token.GetException().Message}");
```

Comment:
This should be NetEventSource.Error intead of .Info.

Reply:
Yes, here should be Error, because the failure will eventually be translated in to a Exception: https://github.com/dotnet/corefx/blob/00bcad0f047186047d3021584b732a8c5d124376/src/System.Net.Security/src/System/Net/Security/SslState.cs#L800 

Will fix.